packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.constant.js

```diff
@@ -41,11 +55,14 @@ export const IP_LOCATION_GROUPS = [
 ];
 
 export const VPS_MAX_QUANTITY = 16;
+export const LEARN_ORGANIZATION_LINK = 'https://docs.ovh.com/fr/';
```

Comment:
It's only for FR ? It isn't adapted for other subsidiary ?

Reply:
Hello Guillaume, This requirement is still in progress. This link is temporary. We will update with new links later.